### ISSUE TITLE
allow DOI to be supplied  for paper curation

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1366]
PHIBase  will  work with journals and the plan is to insist on  curation in canto pre- publication

For this to work, need to be able to supply Doi

(PomBase require this also)

not urgent

**Comment by ValWood** [comment]
Currently,  we can only supply PubMed IDs  "PMID: 24574118"
 
https://www.ncbi.nlm.nih.gov/pubmed/ 
There is some sort of look up which retrieves publication detaisl from PubMed (title, authors, abstract etc)

A paper is only assigned a PubMed ID post-publication.
PHIBase would like to trial "pre-publication" curation  with some of the plant journals.

Therefore we need to be able to supply a  DOI:  http://www.doi.org/
and hopefully retrieve details by a similar method?

(This should be an option that any future  installer could configure at set -up)

**Comment by kimrutherford** [comment]
Now I've had a bit of a think about this.

> Therefore we need to be able to supply a DOI: http://www.doi.org/
> and hopefully retrieve details by a similar method?

That's difficult because as far as I can tell there isn't a way to query for the publication details given a DOI.  That's not unreasonable because DOI is a redirection/indirection service and not every DOI ID resolves to a publication.  For example, here's an DOI for an earthquake: http://dx.doi.org/10.1594/GFZ.GEOFON.gfz2009kciu

We can get around this by requiring the curator to enter the publication title, author list and abstract when they enter the DOI.

Something else that needs thought is that the pre-publication DOI will be different to the DOI of the final publication. Maybe we need to let the curator change the DOI?

**Comment by ValWood** [comment]
Yes! I remember Steve and Duygu had a massive problem with a journal recently because they (the journal)  published an out of date DOI online  for some software and then refused to change it, saying that they could not alter a publication (it was correct in paper and wrong on-line). Eventually Steve had to read the riot act with the journal editor and it did get fixed, but yes we need to be careful, because they aren't  permanent.  Allowing the curator to edit the DOI when the PMID is assigned could be an option.

**Comment by mah11** [comment]
Am I right is assuming the various DOIs for a publication will be connected to the same Canto db record?

Also, it's probably worth keeping in mind that PHI-Base is more likely than PomBase to occasionally curate papers that don't get into PubMed, so they may have some that are always identified by DOI.

**Comment by kimrutherford** [comment]
> Am I right is assuming the various DOIs for a publication will be connected to the same Canto db record?

Yep.  If we allow the publication ID (either DOI or PMID) to be changed we should keep a record of the older IDs.

> so they may have some that are always identified by DOI

I think that should be OK.

**Comment by kimrutherford** [comment]
> That's difficult because as far as I can tell there isn't a way to query for the publication details
> given a DOI.

I think I'm wrong.  :-)

See "Content Negotiation" here: https://www.doi.org/doi_handbook/5_Applications.html#5.4.1
